### ISSUE TITLE
Suggestion: make it a complete finance app by including a good open-source budget app

### ISSUE THREAD

**Issue by maelp** [afadil/wealthfolio#113]
There are many open-source budgeting app like Firefly III, Actual budget, or https://github.com/budgetzero/budgetzero, might be worth it to connect it to the Wealthfolio interface to have a unified view of budget and networth?

**Comment by afadil** [comment]
V3.5 has budgeting and Spending now. 3 Dashboards: Investments, Net Worth and Spending

<img width="1920" height="1423" alt="Image" src="https://github.com/user-attachments/assets/5d0cec32-fae1-4336-97d6-0861a80b45a7" />

<img width="1920" height="1423" alt="Image" src="https://github.com/user-attachments/assets/bce2081f-0e05-491a-9b0a-caeb4d168949" />

<img width="1920" height="1423" alt="Image" src="https://github.com/user-attachments/assets/21ac56b9-1622-4374-ba17-6448a6f37d9f" />

**Comment by maelp** [comment]
Really cool! Would it be possible to have a binding with some kind of bank API so we get transactions directly from the bank account? Or at least import them from CSV?

**Comment by afadil** [comment]
Import CSV is already there and works well with credit card and cash account transactions.

I'm planning to add bank account sync to [Wealthfolio Connect](https://wealthfolio.app/connect/) later, in addition to brokerage sync.

**Comment by Jonjon-prog** [comment]
Hey @afadil, 

Regarding the Wealthfolio connect, do you have any news on the European bank connection ? As it seems more complicated to connect those ones. ?

**Comment by afadil** [comment]
@Jonjon-prog Yeah, bank integration is a whole mess; Connect is using Snaptrade under the hood and depends on their roadmap. I'm exploring Plaid for bank accounts and credit cards, but this is going to take a while.

**Comment by Jonjon-prog** [comment]
@afadil 
Yeah I know.. It's really a pain, and a lot of bank don't expose their api or have really good things about it. 
I know that for other platforms are passing by Powens, but I don't know their price, it's seems free actors in Europe is almost impossible. And I am not sure the American providers got access to the European market too. 
It's really sad people can't access to it securely freely as it's their own data.

**Comment by oliviertassinari** [comment]
> do you have any news on the European bank connection ?

For France, I believe that http://powens.com/ is best in class for the data source.
I currently use it with Google Apps Script (it's free for test individual accounts) to sync everything on a custom Google Sheets built by Claude Code. It works really well.
